web-console/developer/pipelines/pipelines.md

```diff
@@ -1,5 +1,314 @@
 # Pipelines
-The Pipelines page is a primary nav item in the OpenShift Developer perspective.
+The Pipelines page is a primary navigation item in the OpenShift Developer perspective.  Pipelines is only shown when the operator is installed.
+
+## Pipelines Resource Table
+![Pipelines Resource Table](img/PL-ListView.png)
+
+### No projects exist
+When no project exist, clicking Pipelines will show an empty state:
+
+### All Projects in project selector
+When a project is not in context, clicking Pipelines will prompt the user to select from a list of projects:
+![Pipelines Resource Table - All Projects](img/PL-ListView-AllProjects.png)
+
+
+Dev Perspective - Pipelines
```

Comment:
I think this block of text would be great to sprinkle in among the images above to follow along as people are reading the documentation.